tests/test_dijkstra.py

```diff
@@ -39,6 +39,240 @@ def test_dijkstra(self):
         expected = {4: 23.0}
         self.assertEqual(expected, path)
 
+    def test_dijkstra_path(self):
+        g = retworkx.PyDiGraph()
+        a = g.add_node("A")
+        b = g.add_node("B")
+        c = g.add_node("C")
+        d = g.add_node("D")
+        e = g.add_node("E")
+        f = g.add_node("F")
+        edge_list = [
+            (a, b, 7),
+            (c, a, 9),
+            (a, d, 14),
+            (b, c, 10),
+            (d, c, 2),
+            (d, e, 9),
+            (b, f, 15),
+            (c, f, 11),
+            (e, f, 6),
+        ]
+        g.add_edges_from(edge_list)
+        paths = retworkx.digraph_dijkstra_shortest_paths(g, a)
+        expected = {
+            1: [0, 1],
+            2: [0, 3, 2],
+            3: [0, 3],
+            4: [0, 3, 4],
+            5: [0, 1, 5],
```

Comment:
```suggestion
        expected = {
            # a -> b
            1: [0, 1],
            # a -> c: a, d, c
            2: [0, 3, 2],
            # a -> d
            3: [0, 3],
            # a -> e: a, d, e
            4: [0, 3, 4],
            # a -> f: a, b, f
            5: [0, 1, 5],
```

Reply:
This helped me review, up to you if you want to add it. My alpha->zero indexed numeric mental conversion isn't that sharp